api/utils.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"context"
+
+	"github.com/filecoin-project/lotus/chain/address"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+type SignFunc = func(context.Context, []byte) (*types.Signature, error)
```

Comment:
Can this not be an alias also?

Reply:
Using type created import loop with actors